index.bs

```diff
@@ -4983,6 +5129,12 @@ IANA "WebAuthn Extension Identifier" registry established by [[!WebAuthn-Registr
     The user verification method extension returns to the [=[WRP]=] which user verification methods (factors) were
     used for the WebAuthn operation.
 - Specification Document: Section [[#sctn-uvm-extension]] of this specification
+    <br/><br/>
+- WebAuthn Extension Identifier: credProps
+- Description: This [=registration extension=] and [=authentication extension=] enables reporting on credential properties.
+    The credential properties extension returns to the [=[WRP]=] information about the credential used for the WebAuthn operation.
```

Comment:
this is only a registration extension, yes?

suggest:

- Description: This [=registration extension=] enables reporting of a newly-created [=credential=]'s properties to the calling [=[WRP]=] [=web application=].

Reply:
Good pickup. This was original defined as both registration and authentication extension. Now fixed.